app/components/AllPlatformsSettings.js

```diff
@@ -0,0 +1,34 @@
+const React = require('react');
+const store = require('../store');
+const { SUPPORTED_PLATFORMS } = require('../constants');
+const PlatformSetting = require('./PlatformSetting');
+
+const AllPlatformsSettings = React.createClass({
+    onClickHandler() {
+        SUPPORTED_PLATFORMS.forEach(platform => {
+            store.disablePlatform(platform);
+            ga('send', 'event', 'Platform Setting', 'UnCheck', platform);
+        });
+        this.forceUpdate();s
```

Comment:
What's the s at the end?

Reply:
Apparently I wanted to Ctrl+S but my Ctrl is kinda broken